Makefile

```diff
@@ -18,7 +18,7 @@ test_build: build
 	docker-compose -f docker-compose-test.yml build
 
 test: test_build
-	docker-compose -f docker-compose-test.yml run app sh -c "/app/bin/wait-for-it.sh db:5432 -- pytest --cov --cov-report term-missing"
+	docker-compose -f docker-compose-test.yml run app sh -c "/app/bin/wait-for-it.sh db:5432 -- pytest -vvvv --cov --cov-report term-missing"
```

Comment:
Was this for your local testing? Should we be committing this for everybody?

Reply:
it was for my local testing but probably good for everyone. maybe it should be in a different commit/pr